contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1740,7 +1739,7 @@ SELECT
   , CAST(0 as sys.bit)  AS uses_database_collation
   , CAST(0 as sys.bit)  AS is_recompiled
   , CAST(
-      CASE WHEN ao.type IN ('P', 'FN', 'IN', 'TF', 'RF', 'IF') THEN
```

Comment:
Is this changes is acc. to TSQL behaviour? If yes then can we please add a positive testcase as well.

Reply:
This was meant only for functions. Removed it in the next commit.